ansible/roles/ocp4-workload-integreatly/defaults/main.yml

```diff
@@ -3,6 +3,19 @@ become_override: False
 ocp_username: opentlc-mgr
 silent: False
 
+# variables for the minio operator deployment
+minio_operator_ns: "redhat-rhmi-minio-operator"
```

Comment:
If these are part of the Integreatly workload would you mind sticking these variables into the dictionary?
```
ocp4_workload_integreatly_defaults:
  minio_operator_ns
  ...
```

Reply:
But really I'd stick that into its own workload. Something like ocp4-workload-integreatly-minio. And then run that before running the integreatly workload.

Reply:
Sure, thanks for the feedback. We've updated this now to a separate workload